### ISSUE TITLE
Problem with fs-Date when providing date as string (proposition of solution)

### ISSUE THREAD

**Issue by 4rlekin** [formstamp/formstamp#53]
Well im using formstamp having my backend based on Django and Django Rest Framework.

I had this weird problem with fs-Date, to be exact in this method:

    exports.updateDate = function(date, newDate) {
	  switch (false) {
	    case !(date == null):
	      return newDate;
	    case !(newDate == null):
	      return date;
	    default:
	      date.setHours(newDate.getHours());
	      date.setMinutes(newDate.getMinutes());
	      date.setSeconds(newDate.getSeconds());
	      return date;
	  }
	};

I was given errors about setHours (or getHours) not being a function.
It was caused by 'date' variable being string with date and not js Date object.
I fixed it by adding this line:

    date = new Date(date);

at the beginning of 'default' section of switch.
I cannot imagine reason why this shouldn't be there in the first place, so please either include or explain to me why it would be bad (there might be issues with compatibility or usage that i didn't think of).

Great lib anyway :)

**Comment by jrbecart** [comment]
You're right. I did almost the same for fs-datetime. And yes, why this shouldn't be there in the first place???
